default/generated/azure/36-azure-gcp.yaml

```diff
@@ -3,42 +3,40 @@
   description: Google Cloud Pub/Sub usage detected
   effort: 3
   labels:
-  - source=google-pubsub
+  - source=google-cloud-pubsub
   - target=azure-appservice
   - target=azure-aks
   - target=azure-container-apps
   - domain=cloud-readiness
-  - category=messaging
+  - category=messaging-service-migration
   - os=windows
   - os=linux
   links:
-  - title: Azure for Google Cloud Professionals
-    url: https://learn.microsoft.com/en-us/azure/architecture/gcp-professional/services#messaging-and-eventing
-  - title: Azure Service Bus Java samples
-    url: https://learn.microsoft.com/en-us/azure/service-bus-messaging/service-bus-samples
+  - title: Use the Azure portal to create a Service Bus topic and subscriptions to the topic
```

Comment:
same here for the portal doc, and we can add the spring doc for reference: https://learn.microsoft.com/azure/developer/java/spring-framework/using-service-bus-in-spring-applications